docs/source/specs/typespec/main.tsp

```diff
@@ -140,9 +140,52 @@ namespace Workspaces {
         statusCode: 204,
     }
 
+    model PaginationMeta {
+        @doc("Total count of returned objects")
+        @example(10)
+        count: int64;
+
+        @doc("Limit of returned objects")
+        @example(10)
+        limit: int64 = 10;
+
+        @doc("Offset of returned objects")
+        @example(10)
+        offset: int64;
+    }
+
+    model PaginationLinks {
+        @doc("The first link in pagination")
+        @example("/api/rbac/v2/(resource)/?limit=10&offset=0")
+        first: string;
```

Comment:
Maybe adding `@format("uri")` for the `PaginationLinks` fields?

Reply:
Updated!